UI/style-guest.css

```diff
@@ -0,0 +1,349 @@
+@font-face {
+    font-family: 'Open Sans';
+    font-style: normal;
+    font-weight: 400;
+    src: local(''),
+         url('../fonts/open-sans-v18-latin-regular.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
+         url('../fonts/open-sans-v18-latin-regular.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+  }
+
+  html {
+      box-sizing: border-box;
+  }
+  *, *::before, *::after {
+      box-sizing: inherit;
+  }
+  /*start task*/
+
+.container {
+    width: 1920px;
+     background-color: honeydew;
+}
+.wrapper {
+    width : 1700px;
+    margin: 0 auto;
+}
+header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.header__nav-list {
+    width: 1300px;
+    display: flex;
+    justify-content: space-between;
+    font-size: 35px;
+    align-items: center;
+    border-bottom: 3px solid black;
+}
+.logo {
+    width: 250px;
+}
+a {
+    text-decoration: none;
+    color: black;
+}
+.add-adv {
+    width: 350px;
+    height: 60px;
+    padding-top: 15px;
+    text-align: center;
+    background-color: powderblue;
+}
+.user-block {
+    display: flex;
+    align-items: center;
+}
+.user-guest {
+    color: blue;
+    text-decoration: underline;
+}
+.user-list {
+    width: 300px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+.user-logo {
+    width: 60px;
+}
+/*search*/
+
+.search-block {
+    width: 1500px;
+    height: 300px;
+    margin: 0 auto;
+    border: 2px solid black;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    font-size: 25px;
+    background-color: lightblue;
+}
+.search-btn {
+    width: 120px;
+    height: 60px;
+    font-size: 25px;
+    background-color: powderblue;
+}
+/* adv-block*/
+
+.adve {
+    width: 1500px;
+    margin: 0 auto;
+    margin-top: 70px;
+    display: flex;
+    font-size: 25px;
+    height: 250px;
+}
+.adve-description {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-left: 30px;
+}
+.headlines {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.adve-name {
+    font-size: 45px;
+}
+.remont-logo {
+    width: 250px;
+    height: 250px;
+}
+.adve-href {
+    color: cornflowerblue;
+    text-decoration: underline;
+}
+.sell {
+    color: red;
+    font-size: 35px;
+}
+.end {
+    display: flex;
+    justify-content: space-between;
+}
+.rate {
+    font-size: 30px;
+    color: royalblue;
+}
+.name-rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.comments {
+    width: 1500px;
+    margin: 0 auto;
+}
+.com {
+    font-size: 25px;
+    border: 2px solid black;
+    border-radius: 5px;
+    padding: 3px;
+    background-color: mintcream;
+}
+.mark {
+    font-size: 35px;
+    padding-right: 15px;
+    padding-left: 15px;
+}
+.user-com {
+    width: 60px;
+    height: 50px;
+}
+.com-1, .com-2, .com-3 {
```

Comment:
Is there any reason to have 3 different classes for the same purpose with the same styles?